From 2b3f7e4fdba90682476fa3a804bbfc8dc550e445 Mon Sep 17 00:00:00 2001
From: Mugunthan V N <mugunthanvnm@ti.com>
Date: Tue, 5 Jun 2012 13:37:47 +0530
Subject: [PATCH 1030/1092] ARM: OMAP2+: AM33XX: PM: remove cpsw hack for pm

Removing cpsw hack to disable the module in suspend as cpsw supports rumtime pm

Signed-off-by: Mugunthan V N <mugunthanvnm@ti.com>
---
 arch/arm/mach-omap2/pm33xx.c | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/arch/arm/mach-omap2/pm33xx.c b/arch/arm/mach-omap2/pm33xx.c
index 47dfa46..46b89d5 100644
--- a/arch/arm/mach-omap2/pm33xx.c
+++ b/arch/arm/mach-omap2/pm33xx.c
@@ -115,17 +115,14 @@ static int am33xx_pm_suspend(void)
 {
 	int state, ret = 0;
 
-	struct omap_hwmod *cpgmac_oh, *gpmc_oh, *usb_oh;
+	struct omap_hwmod *gpmc_oh, *usb_oh;
 
-	cpgmac_oh	= omap_hwmod_lookup("cpgmac0");
 	usb_oh		= omap_hwmod_lookup("usb_otg_hs");
 	gpmc_oh		= omap_hwmod_lookup("gpmc");
 
-	omap_hwmod_enable(cpgmac_oh);
 	omap_hwmod_enable(usb_oh);
 	omap_hwmod_enable(gpmc_oh);
 
-	omap_hwmod_idle(cpgmac_oh);
 	omap_hwmod_idle(usb_oh);
 	omap_hwmod_idle(gpmc_oh);
 
-- 
1.7.11.2

